data/json/mapgen/house08.json

```diff
@@ -85,21 +85,21 @@
                 { "point" : "terrain", "id" : "t_tree",       "x" : [ 2,14], "y" : [17,21], "repeat" : [2, 6] },
                 { "point" : "terrain", "id" : "t_tree_young", "x" : [ 2,14], "y" : [17,21], "repeat" : [3, 5] }
             ],
-            "place_items" : [
-                { "item" : "bed",        "chance" : 80, "x" : [ 4, 5], "y" : [ 7, 8] },
-                { "item" : "bedroom",    "chance" : 70, "x" : [ 2, 8], "y" : [ 6, 8] },
-                { "item" : "livingroom", "chance" : 80, "x" : [ 9,21], "y" : [ 2, 7] },
-                { "item" : "homeguns",   "chance" : 20, "x" : [ 2, 2], "y" : [10,11] },
-                { "item" : "dresser",    "chance" : 80, "x" : [ 2, 2], "y" : [10,11] },
-                { "item" : "dresser",    "chance" : 50, "x" : [ 2, 2], "y" : [ 7, 7] },
-                { "item" : "fridge",     "chance" : 50, "x" : [19,19], "y" : [11,11] },
-                { "item" : "kitchen",    "chance" : 50, "x" : [21,21], "y" : [11,11] },
-                { "item" : "kitchen",    "chance" : 50, "x" : [18,18], "y" : [11,11] },
-                { "item" : "kitchen",    "chance" : 70, "x" : [18,20], "y" : [ 7,10] },
-                { "item" : "oven",       "chance" : 50, "x" : [21,21], "y" : [10,10] },
-                { "item" : "cleaning",   "chance" : 50, "x" : [ 2, 5], "y" : [13,14] },
-                { "item" : "softdrugs",  "chance" : 70, "x" : [ 2, 5], "y" : [12,14] },
-                { "item" : "office",     "chance" : 50, "x" : [ 2, 7], "y" : [ 2, 4] }
+            "place_loot" : [
+                { "group" : "bed",        "chance" : 90, "x" : [ 4, 5], "y" : [ 7, 8], "repeat" : [1,4] },
+                { "group" : "bedroom",    "chance" : 80, "x" : [ 2, 8], "y" : [ 6, 8], "repeat" : [1,6] },
```

Comment:
I'd go down with most numbers by about 30%. The best way to think about it is that a house should contain (on average) no more than a dozen items or so